app/lang/zh-TW.json

```diff
@@ -1,925 +1,892 @@
 {
-    "\"Launch on system start\" only works in XDG-compliant desktop environments. To enable the Mailspring icon in the system tray, you may need to install libappindicator.": "「系統啟動時啟動」僅適用於符合 XDG 標準的桌面環境。要在系統託盤中啟用 Mailspring 圖示，您可能需要安裝 libappindicator。",
```

Comment:
This nearly two-thousand-line diff is extremely risky and has indeed caused issues here.